app/controllers/calculators_controller.rb

```diff
@@ -2,4 +2,10 @@
 
 class CalculatorsController < ApplicationController
   def show; end
+
+  private
+
+  def set_calculator
+    @calculator = Calculator.friendly.find(params[:id])
+  end
```

Comment:
This method is not used. 
1. Rollback to the previous version with calculator retrieving in `show` action
2. Add migration to create a new calculator record with the name `Diapers calculator`
3. In the feature test create a new calculator record using factory

Reply:
Thanks, done)